services/sidecar/Dockerfile

```diff
@@ -119,7 +119,7 @@ HEALTHCHECK --interval=60s \
   --timeout=60s \
   --start-period=30s \
   --retries=3 \
-  CMD celery inspect ping -b amqp://${RABBIT_USER}:${RABBIT_PASSWORD}@${RABBIT_HOST}:${RABBIT_PORT} -d celery@${HOSTNAME}
+  CMD celery inspect ping -b amqp://${RABBIT_USER}:${RABBIT_PASSWORD}@${RABBIT_HOST}:${RABBIT_PORT} -d sidecar@${HOSTNAME}
```

Comment:
thisone was not pointing to the right service? 
Why not adding ``SIDECAR_HOST_HOSTNAME_PATH`` like in dev mode?

Reply:
good catch. will check

Reply:
done